src/vmm/src/lib.rs

```diff
@@ -98,6 +98,9 @@ pub const FC_EXIT_CODE_SIGILL: ExitCode = 157;
 pub const FC_EXIT_CODE_BAD_CONFIGURATION: ExitCode = 152;
 /// Command line arguments parsing error.
 pub const FC_EXIT_CODE_ARG_PARSING: ExitCode = 153;
+// Received Data Available interrupt - for letting the driver know that
+// there is some pending data to be processed.
+const IER_RDA_BIT: u8 = 0b0000_0001;
```

Comment:
This would be more suited in a serial-specific file (maybe `src/devices/src/legacy/serial.rs`). Anyone looking at the main `vmm` file would have no idea what `IER_RDA_BIT` means.